pallets/handles/src/tests/signed_extension_tests.rs

```diff
@@ -0,0 +1,36 @@
+use crate::{handles_signed_extension::HandlesSignedExtension, tests::mock::*};
+use frame_support::{assert_ok, dispatch::DispatchInfo};
+use sp_core::{sr25519, Pair};
+use sp_runtime::traits::SignedExtension;
+
+/// Assert that retiring a handle passes the signed extension HandlesSignedExtension
+#[test]
+fn signed_extension_retire_handle_success() {
+	new_test_ext().execute_with(|| {
+		let base_handle_str = "test1";
+
+		// Claim the handle
+		let alice = sr25519::Pair::from_seed(&[0; 32]);
+		let (payload, proof) =
+			get_signed_claims_payload(&alice, base_handle_str.as_bytes().to_vec());
+		assert_ok!(Handles::claim_handle(
+			RuntimeOrigin::signed(alice.public().into()),
```

Comment:
Why is it necessary to create keys here? We do not do this in any other mock file.